tests/test_osx_utils.sh

```diff
@@ -45,7 +57,14 @@
 [ "$(macpython_impl_for_version pypy-5.4)" == "pp" ] || ingest
 
 # Test lookup of available macOS SDK build targets from python version
-[ "$(macpython_sdk_list_for_version 3.8)"    == "10.9" ] || ingest
+[ "$(macpython_sdk_list_for_version 3.11)" == "11.0" ] || ingest
+[ "$(macpython_sdk_list_for_version 3.11.3)" == "11.0" ] || ingest
+[ "$(macpython_sdk_list_for_version 3.9)"    == "10.9 11.0" ] || ingest
+[ "$(macpython_sdk_list_for_version 3.9.3)"  == "10.9 11.0" ] || ingest
+[ "$(macpython_sdk_list_for_version 3.9.0)"  == "10.9" ] || ingest
+[ "$(macpython_sdk_list_for_version 3.8)"    == "10.9 11.0" ] || ingest
+[ "$(macpython_sdk_list_for_version 3.8.10)"  == "10.9 11.0" ] || ingest
+[ "$(macpython_sdk_list_for_version 3.8.3)"    == "10.9" ] || ingest
```

Comment:
```suggestion
[ "$(macpython_sdk_list_for_version 3.8.10)" == "10.9 11.0" ] || ingest
[ "$(macpython_sdk_list_for_version 3.8.3)"  == "10.9" ] || ingest
```
In devel at the moment, this block attempts to vertically line up the equality checks.